modules/administration-guide/pages/configuring-network-policies.adoc

```diff
@@ -105,4 +105,4 @@ The default is `{prod-namespace}`.
 
 * link:https://kubernetes.io/docs/concepts/security/multi-tenancy/#network-isolation[Network isolation]
 
-* link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp4-ver}/html/networking/network-security#nw-networkpolicy-multitenant-isolation_multitenant-network-policy[Configuring multitenant isolation with network policy]
+* link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp4-ver}/html/network_security/network-policy#multitenant-network-policy[Configuring multitenant isolation with network policy]
```

Comment:
Does `{ocp4-ver}` need to be replaced in this line too?

Reply:
I'm a silly goose. it's the other way round, both times, i need to replace the ocp version with an attribute. it's fixed now